sharedproject.props

```diff
@@ -8,6 +8,7 @@
     <PackageProjectUrl>https://steeltoe.io</PackageProjectUrl>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <RunAnalyzersDuringLiveAnalysis>false</RunAnalyzersDuringLiveAnalysis>
```

Comment:
I'm not necessarily opposed, but why do you want to turn this off here?

Reply:
Ahh .. mistakenly left in .. I'll remove